### ISSUE TITLE
vesktop: rich presence does not work in celeste

### ISSUE THREAD

**Issue by NyCodeGHG** [NixOS/nixpkgs#325121]
### Describe the bug

The discord rich presence of vesktop does not work when playing [Celeste](https://www.celestegame.com/) with [Everest](https://everestapi.github.io).

### Steps To Reproduce
1. Run Vesktop
2. Run Celeste with Everest installed
3. There is no rich presence

### Expected behavior

Everest's rich presence should be visible.

### Additional context

this shows up in Celeste's console log:
```
[Everest] [Error] [discord-game-sdk] Ignoring response to unknown nonce: IncomingMessage { cmd: "DISPATCH", nonce: Some("initial-ready"), evt: Some("READY"), data: Object({"config": Object({"api_endpoint": String("//discord.com/api"), "cdn_host": String("cdn.discordapp.com"), "environment": String("production")}), "user": Object({"avatar": String("cfefa4d9839fb4bdf030f91c2a13e95c"), "discriminator": String("0000"), "flags": Number(0), "id": String("1045800378228281345"), "premium_type": Number(0), "username": String("arRPC")}), "v": Number(1)}) }
```

This error message does not show up when using the [official vesktop flatpak](https://flathub.org/apps/dev.vencord.Vesktop), so it looks like it's a NixOS issue. (The rich presence works when using the flatpak)

### Notify maintainers

@getchoo
@Scrumplex 
@vgskye 
@pluiedev 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.9.6-zen1, NixOS, 24.11 (Vicuna), 24.11.20240627.b2852eb`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.4`
 - nixpkgs: `/nix/store/xjn3184743q0fxayzwrhil2i06pr61x8-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by NotAShelf** [comment]
This sounds like a issue that you should report to Vencord or arRPC (assuming you use that for RPC) developers. Packaging of Vencord should minimally affect its functionality, if at all.

**Comment by NyCodeGHG** [comment]
looks like it got fixed in 1.5.3, i made a pr to update: #325137
